lib/rack/response.rb

```diff
@@ -38,16 +38,28 @@ def header
       headers
     end
 
+    # :call-seq:
+    #   Response.new(body = nil, status = 200, headers = {}) => response
```

Comment:
I personally don't see the value of the call-seq here.

Reply:
Okay, removed.